### ISSUE TITLE
Color Palette

### ISSUE THREAD

**Issue by MelSumner** [emberjs/website#3146]
Goal: Determine the correct color palette for the website

Considerations: 
- interactive elements should look interactive
- Must include:
-- Ember orange (E04E39)
-- White (#fdfdfd) & Black (#212121)
- must pass color contrast tests (use https://webaim.org/resources/contrastchecker/ or similar)

**Comment by kennethlarsen** [comment]
Is the aim to pass WCAG AA or WCAG AAA in terms of color contrast?

**Comment by MelSumner** [comment]
@kennethlarsen AA

**Comment by amyrlam** [comment]
Let's put this improvement in https://github.com/ember-learn/ember-styleguide instead, per the conversation on https://github.com/emberjs/website/pull/3613. I updated the issue summary above. 

Thanks @ctcpip for getting this started! 😄
